content/07-native-tokens/03-getting-started.mdx

```diff
@@ -445,17 +429,17 @@ Use this code to calculate the minimum fee for the transaction:
 --tx-in-count 1 \
 --tx-out-count 2 \
 --witness-count 1 \
---testnet-magic 3 \
+--testnet-magic 1097911063 \
 --protocol-params-file protocol.json
 
 178393 Lovelace
 
 ./cardano-cli transaction build-raw \
-	     --mary-era \
+	     --babbage-era \
```

Comment:
We could remove all instances of `--babbage-era` in this document because the cli will soon default to current era. If anyone tried this before then an error will ask them to include this flag.